.github/workflows/network-deploy.yml

```diff
@@ -82,8 +97,10 @@ jobs:
             -var="values_file=${{ env.VALUES_FILE }}" \
             -var="gke_cluster_context=${{ env.GKE_CLUSTER_CONTEXT }}" \
             -var="aztec_docker_image=${{ env.AZTEC_DOCKER_IMAGE }}" \
-            -out=tfplan
+            -var="l1_deployment_mnemonic=${{ steps.get-mnemonic.outputs.mnemonic }}" \
+            -out=tfplan \
+            -lock=${{ inputs.respect_tf_lock }}
```

Comment:
this is to avoid concurrent updates, right? not against it, curious if we think there's a chance of that happening

Reply:
Yes, and I don't know. Two people could click through the UI at the same time so I'd rather not risk it.